workspace/__lib__/xod/gpio/change-interrupt/patch.xodp

```diff
@@ -0,0 +1,71 @@
+{
+  "description": "Attaches the interrupt for the specified port in the CHANGE mode: triggers whenever the pin changes value. Interrupt increments a counter and store the time in microseconds of the latest interrupt. The node will emit these values and pulse as soon as possible, but not right in the moment when an interrupt occurred.",
+  "nodes": [
+    {
+      "description": "Board port to attach interrupt to.",
```

Comment:
```suggestion
      "description": "Board port to attach interrupt to. Note that not all pins support hardware interrupts. Refer to your board specs to learn which are compatible.",
```

Also, we should tell here what will happen if the port does not support interrupts.

Reply:
I'll add a `static_assert` so in this case, it will be a compile-time error.